pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -118,7 +118,7 @@ let
         ] ++ optionals x11Support [ tcl tk xlibsWrapper libX11 ]
     )
     ++ optional zlibSupport zlib
-    ++ optional stdenv.isDarwin CF;
+    ++ optionals stdenv.isDarwin [ coreutils CF ];
```

Comment:
I personally prefer to be as explicit as possible with `buildInputs` and `propagatedBuildInputs` in order to prevent situations like https://github.com/NixOS/nixpkgs/issues/12300 but its far from common practice in Nixpkgs.